### DISCUSSION TITLE
TTS

### DISCUSSION THREAD

**Discussion by seth225** [open-webui/open-webui#5844] [Q&A]
Ive got Open Webui set up with a docker container.   Also have Openedai=Speech set up with a docker container.   There are  several voices in the 'voice' folder but within the Open Webui settings there is only one voice selection..  alloy.   What I missed something or done something incorrectly?   It seems that the two containers are not interfacing?

**Comment by kendrick90** [comment]
I noticed this too. The drop down is not working correctly. If you delete the word alloy the other names show up for selection. The drop down for the TTS Model also has the same issue but deleting and then selecting one or the other works.

It seemed that when I tried to select a different voice after deleting the text and selecting another from the drop down it always used the alloy voice. I found that after I manually typed in echo, saved, and then refreshed the conversation page it actually switched voices. After that selecting between voices using the drop down worked although you still have to delete the text to see the drop down. And always have to refresh the conversation page to get the new voice. 

It is possible that the manually typing once is not necessary and the real issue was I wasn't refreshing the page to get the new voice but I think I did try refreshing between switches before. If you or someone else who finds this can try and see if the typing in step is necessary and report back here it could help the devs. I tried restarting my containers to see if it showed up again but only the drop down issue remained so I am not sure. Hope this helps.
